vault/external_tests/raft/raft_test.go

```diff
@@ -438,27 +442,48 @@ func TestRaft_Configuration(t *testing.T) {
 	t.Parallel()
 	cluster, _ := raftCluster(t, nil)
 	defer cluster.Cleanup()
+	testRaft_Configuration(t, cluster)
+}
 
-	for i, c := range cluster.Cores {
-		if c.Core.Sealed() {
-			t.Fatalf("failed to unseal core %d", i)
-		}
+func TestRaft_Configuration_Docker(t *testing.T) {
+	t.Parallel()
+	binary := os.Getenv("VAULT_BINARY")
+	if binary == "" {
+		t.Skip("only running docker test when $VAULT_BINARY present")
+	}
+	opts := &docker.DockerClusterOptions{
+		ImageRepo: "hashicorp/vault",
+		// TODO ideally we'd use this branch's latest version, which works
+		// fine on release branches, but on main it might not exist yet.
+		ImageTag:    "1.13.0",
```

Comment:
Out of curiosity, why not just `latest`? Given you're dumping a new Vault binary into the container anyways, I don't know that there's much benefit to different base image versions tbh, assuming we haven't made breaking changes to env vars.

Reply:
Oh, at the time I wrote that I didn't have the t.Skip, and I was thinking for developers running locally, it might be nice to able to run the test without having to configure an env var, even if that meant they wouldn't be testing their actual checked out code.  But I think you're right that latest makes more sense now.